Makefile

```diff
@@ -62,6 +62,23 @@ k8s: validate ## Build default K8s version of EKS Optimized AL2 AMI
 1.23: ## Build EKS Optimized AL2 AMI - K8s 1.23
 	$(MAKE) k8s kubernetes_version=1.23.9 kubernetes_build_date=2022-07-27 pull_cni_from_github=true
 
+# FIPS-mode enabled versions of EKS Optimized AL2 AMIs
+.PHONY: 1.20-fips
+1.20-fips: ## Build EKS Optimized AL2 AMI - K8s 1.20
+	$(MAKE) k8s kubernetes_version=1.20.15 kubernetes_build_date=2022-07-27 pull_cni_from_github=true enable_fips_mode=true
+
+.PHONY: 1.21-fips
+1.21-fips: ## Build EKS Optimized AL2 AMI - K8s 1.21
+	$(MAKE) k8s kubernetes_version=1.21.14 kubernetes_build_date=2022-07-27 pull_cni_from_github=true enable_fips_mode=true
+
+.PHONY: 1.22-fips
+1.22-fips: ## Build EKS Optimized AL2 AMI - K8s 1.22
+	$(MAKE) k8s kubernetes_version=1.22.12 kubernetes_build_date=2022-07-27 pull_cni_from_github=true enable_fips_mode=true
+
+.PHONY: 1.23-fips
+1.23-fips: ## Build EKS Optimized AL2 AMI - K8s 1.23
+	$(MAKE) k8s kubernetes_version=1.23.9 kubernetes_build_date=2022-07-27 pull_cni_from_github=true enable_fips_mode=true
+
```

Comment:
These new make targets don't seem necessary; it's a lot of duplication just to have `make 1.22-fips` instead of `make 1.22 enable_fips_mode=true`.